code/game/machinery/wishgranter.dm

```diff
@@ -24,22 +25,30 @@
 
 	else if(is_special_character(user))
 		user << "Even to a heart as dark as yours, you know nothing good will come of this.  Something instinctual makes you pull away."
+		return
+
+	else if(inuse)
```

Comment:
This else-if chain is completely pointless since all of them just kill the proc.